notebooks/contrib-dev/McCabe_Thiele_Steven_Final.ipynb

```diff
@@ -0,0 +1,1665 @@
+        "x = liqvapx\n",
+        "y = X.dot(beta_hat)\n",
+        "\n",
+        "# create empty lists to store x and y coordinates for the VLE equilibrium graph\n",
+        "x = liqvapx\n",
+        "liqvapy = liqvapy\n",
+        "\n",
+        "def VLE_eq(x):\n",
+        "    \"\"\"plot the \"stair case\" line of the McCabe-Thiele diagram\n",
+        "\n",
+        "    Arguments:\n",
+        "        x: x value on the VLE diagram\n",
+        "\n",
+        "    Returns:\n",
+        "        liqvap: y value on the VLE diagram\n",
+        "\n",
+        "    \"\"\"\n",
+        "    liqvap = (\n",
+        "        beta_hat[0]\n",
+        "        + beta_hat[1] * x\n",
+        "        + beta_hat[2] * x**2\n",
+        "        + beta_hat[3] * x**3\n",
+        "        + beta_hat[4] * x**4\n",
+        "        + beta_hat[5] * x**5\n",
+        "        + beta_hat[6] * x**6\n",
+        "    )\n",
+        "    return liqvap\n",
+        "\n",
+        "\n",
+        "y = VLE_eq(x)\n",
+        "\n",
+        "# Plot the VLE line\n",
+        "fig = plt.figure(figsize=(6, 6))\n",
+        "plt.plot(liqvapx, liqvapy, \"o\", label=\"Data Points\")\n",
+        "plt.plot(x, y, color=\"black\", label=\"Modeled Equation VLE Line\")\n",
+        "plt.plot([0, 1], [0, 1], color=\"orange\", linestyle=\":\", label=\"45° Line\")\n",
+        "plt.xlabel(\"Liquid Mole Fraction of Methanol\", fontsize=10)\n",
+        "plt.ylabel(\"Vapor Mole Fraction of Methanol\", fontsize=10)\n",
+        "plt.xticks(fontsize=10)\n",
+        "plt.yticks(fontsize=10)\n",
+        "plt.tick_params(direction=\"in\", top=True, right=True)\n",
+        "plt.title(\"VLE Equilibrium of Methanol\", fontsize=10)\n",
+        "plt.legend(fontsize=10, bbox_to_anchor=(1.0, 0.15), borderaxespad=0)\n",
+        "plt.show()\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xpHXH5xD-6as"
+      },
+      "source": [
+        "## 3.3. Plot the McCabe-Thiele for a Total Reflux Run"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZiG2JcDWfyPn"
+      },
+      "source": [
+        "## 3.3.a. Stepping line\n",
+        "\n",
+        "The stepping line connects the points that represent the mole fraction of Methanol in each stage of the distillation column. It starts at the bottom mole fraction of Methanol on the 45 degree line and ends after reaching the mole fraction of Methanol in the distillate on the 45 degree line.\n",
+        "\n",
+        "![](../../media/MCabe_thiele_stepping_line.png)\n",
+        "\n",
+        "Create a function that is able to generate the points requried to plot the stepping line."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 9,
+      "metadata": {
+        "id": "mioHfOn4ht0G"
+      },
+      "outputs": [],
+      "source": [
+        "def stair(slope, y_intercept, x_start, y_start, y_end, Efficiency=1):\n",
```

Comment:
Name this function sep_stair or something similar to distinguish from a more general stair plotting function in matplotlib

Reply:
Do not capitalize "Efficiency" as a variable name. Often in Python, capital names mean this is a class.